demosplan/DemosPlanCoreBundle/Controller/Procedure/DemosPlanProcedureController.php

```diff
@@ -1399,7 +1399,17 @@ public function administrationEditAction(
             } else {
                 $template = '@DemosPlanCore/DemosPlanProcedure/administration_edit.html.twig';
                 $title = 'procedure.adjustments';
+
+                $evaluatingPhase = null;
+                foreach ($templateVars['internalPhases'] as $internalPhase) {
```

Comment:
I think we can check for         participationstate: !php/const demosplan\DemosPlanCoreBundle\Entity\Procedure\Procedure::PARTICIPATIONSTATE_FINISHED
        here to keep things dynamically and double check, that this key is set in any procedurephases.yml